integration-tests/tests/src/tests/sync/flexible.ts

```diff
@@ -2075,4 +2081,400 @@ describe("Flexible sync", function () {
       });
     });
   });
+
+  describe("Progress notification", function () {
+    this.timeout(5000);
+    before(async function (this: UserContext) {
+      // TODO: Investigate this.
+      // Use a new user for this test suite as not doing so causes some session issues in other tests
+      this.user = await this.app.logIn(Credentials.anonymous(false));
+    });
+    beforeEach(async function (this: RealmContext & UserContext) {
+      // @ts-expect-error Using an internal API
+      this.realm = new Realm({
+        schema: [Person, Dog],
+        sync: {
+          flexible: true,
+          user: this.user,
+          _sessionStopPolicy: SessionStopPolicy.Immediately,
+        },
+      });
+    });
```

Comment:
Is there a reason you're not using `openRealmBeforeEach` here?

Reply:
Hm, I was moving back and forth between using that or my own as I had other cleanup hooks to check different things but now it seems like this can be fully covered by`openRealmBeforeEach`, feel free to change it.